Applications/DataExplorer/DataView/StationTreeModel.cpp

```diff
@@ -102,6 +102,17 @@ vtkPolyDataAlgorithm* StationTreeModel::vtkSource(const std::string &name) const
     return nullptr;
 }
 
+void StationTreeModel::setNameForItem(const std::string &stn_vec_name, std::size_t id, std::string item_name)
+{
+    auto const stn_list = find_if(_lists.begin(), _lists.end(), [&stn_vec_name](ModelTreeItem* item)
+        { return (stn_vec_name.compare( item->data(0).toString().toStdString() ) == 0); });
+
+    if (id >=  (*stn_list)->childCount())
```

Comment:
If child count is 0 everything still works fine. Also, empty station lists are not added when opening a file so this could not even happen.